source/examples/generated/react-native/CreateToDoPrompt.snippet.create-todo-prompt.tsx

```diff
@@ -0,0 +1,39 @@
+type Props = {
+  onSubmit: ({summary}: {summary: string}, {priority}: {priority: string}) => void;
```

Comment:
Where's the source for this? Anyway, seems weird that you pass two object parameters instead of one. What's going on here? I would have expected:
```suggestion
  onSubmit: (args: {summary: string; priority: string;}) => void;
```

Reply:
Ah, I forgot to link to my [branch with the Bluehawk snippets](https://github.com/krollins-mdb/realm-template-apps/tree/rn-tutorial-v2-bluehawk/sync-todo/v2/client/react-native). The source is `CreateToDoPrompt.tsx` in the template app. The original code only has the summary type:
``` typescript
type Props = {
  onSubmit: ({summary}: {summary: string}) => void;
};
```
I tried to keep the changes minimal for building on top of the template app, so mimicked the existing structure. I haven't seen this args structure before. I think it's cleaner, but it doesn't build. Since I haven't seen this before, I'm not sure how to debug it. Tried a few things with no luck.
```
error: SyntaxError: /Users/kyle.rollins/Documents/GitHub/realm-template-apps/sync-todo/v2/client/react-native/src/CreateToDoPrompt.tsx: Unexpected token, expected ";" (10:47)

   8 | // :snippet-start: create-todo-prompt
   9 | type Props = {
> 10 |   (args: {summary: string; priority: string;}) => void;
     |                                                ^
  11 | };
  12 |
  13 | export function CreateToDoPrompt(props: Props): React.ReactElement<Props>
```